css/src/components/media.scss

```diff
@@ -0,0 +1,41 @@
+.media {
+	display: flex;
+	align-items: flex-start;
+
+	+ .media {
+		border-top: 1px solid $border;
+		margin-block-start: 1rem;
+		padding-block-start: 1rem;
+	}
+
+	.media {
+		display: flex;
```

Comment:
Thanks for pointing this out; this was "inspired" by Bulma may years ago. I consolidate some rules to avoid some duplication. I tested locally with the commented out html on media.md